nais/dev.yaml

```diff
@@ -27,8 +27,7 @@ spec:
   env:
     - name: SPRING_PROFILES_ACTIVE
       value: dev
-  prometheus:
-    enabled: true
-    path: /altinn-rettigheter-proxy/internal/actuator/prometheus
+    - name: REDIS_HOST
```

Comment:
Helst ikke fjern Prometheus, da får vi ikke Grafana-board med logginnsla, alerts++

Reply:
Prometheus is back